docs/api-docs/storefront/widgets/widget-builder.md

```diff
@@ -0,0 +1,187 @@
+- [Install Widget Builder](#install-widget-builder)
+- [Configure Widget Builder](#configure-widget-builder)
+- [Reset or create parallel configurations](#reset-or-create-parallel-configurations)
+- [Generate a scaffold with `create`](#generate-a-scaffold-with-create)
+- [Continue development with `start`](#continue-development-with-start)
+- [Publish to store](#publish-to-store)
+- [Resources](#resources)
+
+</div>
+
+Widget Builder is a command-line tool that lets you build, edit, and preview custom storefront widgets in real time, outside the context of your BigCommerce store.
+
+This article is a guide to help you learn how to use Widget Builder. 
+
+## Prerequisites
+
+* Node.js 14
+* [npm](https://www.npmjs.com/) 6.14
+* A BigCommerce [store](https://support.bigcommerce.com/s/article/Starting-a-Bigcommerce-Trial) or [sandbox store](https://developer.bigcommerce.com/api-docs/partner/getting-started/create-a-sandbox-store).
+* API `access_token` with `content modify` scope. For information on how to generate store API credentials, see [Obtaining store API credentials](https://developer.bigcommerce.com/api-docs/getting-started/authentication/rest-api-authentication#obtaining-store-api-credentials).
+
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+<!-- theme: info -->
+
+> Refer to the [Widget Builder README.md](https://github.com/bigcommerce/widget-builder) for the latest supported `node` version.
+
+</div>
+</div>
+</div>
+
+## Install Widget Builder
+
+Follow these instructions to install Widget Builder and its dependencies. 
+
+1. In a terminal, clone the [widget-builder](https://github.com/bigcommerce/widget-builder) GitHub repo.
+
+2. Navigate to the `widget-builder` directory and install Widget Builder.
+
+```shell
+npm run install-cli
+```
+
+3. To retrieve usage information and view supported options and commands, run the following command:
+
+```shell
+widget-builder -h
+```
+
+**Sample output:**
+
+```shell
+Usage: widget-builder [options] [command]
+
+Options:
+  -V, --version                  output the version number
+  -h, --help                     display help for command
+
+Commands:
+  init                           Initialization of widget builder configuration
+  start [options] [widgetPath]   starts the widget builder locally
+  validate [options] <file>
+  create <widget-template-name>  Create a blank widget template
+  publish <widget-template>      Releases the widget template to the store
+                                 belonging to the env config
+  help [command]                 display help for command
+```
+
+## Configure Widget Builder
+
+Once you have installed Widget Builder, the next step is to set up the widget development environment for your store.  This will authorize you to access your store's data during the development process. You will need the store's `Client ID`, `Access Token`, and `API Path` to generate the appropriate configuration. 
```

Comment:
Once you have installed Widget Builder, the next step is to set up the widget development environment for your store.  This setup will authorize you to access your store's data during the development process. You will need the store's `Client ID`, `Access Token`, and `API Path` to generate the appropriate configuration.

Reply:
Revisions in [commit 2bc1188](https://github.com/bigcommerce/dev-docs/pull/1200/commits/2bc1188132148dfdbf8190f63b162a853636ac7b) address these concerns.